### ISSUE TITLE
Improve the flow for adding new page blocks below existing stories

### ISSUE THREAD

**Issue by alcurrie** [Automattic/amp-wp#1532]
**As a AMP story author, I want a better user flow for adding new page blocks below existing stories, so I'm able to easily understand how to add new pages to my AMP story**
 - [ ] _AC 1:_ Identify interface changes from what is currently available for blocks in AMP stores, that would improve the flow to add new story pages below existing story pages (i.e. show 'ghosted' page with icon)
          - [ ] Display ghosted Story page by default on a new AMP story or below the last AMP story page as shown in this mockup: https://cl.ly/4840a3512886
          - [ ] On tap/click anywhere on the 'ghosted story' page a new story page is created
          - [ ] When a new story page is created,  the block navigation with the (+) Add Layer icon above the Block Navigation (heading)  and the Story page top level on the block navigation selected
![initialstateblocknavafterpagecreation](https://user-images.githubusercontent.com/18579988/48520951-929b4380-e840-11e8-9c24-0586c22d011c.png)
 - [ ] _AC 2:_ Add page number to the AMP stories page interface to represent what story page the user currently editing.   (_Based on the new, easier story page/layer  manipulation #1564 consider not numbering story pages in order to not cause confusion when pages are 'reordered'._) 
- [ ] _AC 3:_ Update the page icon in the block navigation active story to be aligned with/match the updated ‘add a new page’  icon  https://cl.ly/24a5c2e8ab6e

**Comment by westonruter** [comment]
Note: The big plus icon now seems to be part of Gutenberg itself, although it is not shown in a ghosted page:

![image](https://user-images.githubusercontent.com/134745/47463402-f8912f80-d79a-11e8-8218-f74840266f8a.png)

**Comment by alcurrie** [comment]
@westonruter Thanks for the call out on the plus already being available. I've updated AC1 to take into account that this task maybe partially done, though may need some additional improvements to meet the overall 'goal' in this ticket. 

@jwold -  Based on Weston's screen shot, this looks like we'll need to take this into account when making this interface change to make adding pages easier. 
 From the screen shot Weston shared,  as a user, I'd assume the plus at the top of (inside) the card would add a new layer, and the plus below the card would add a new page. Assuming the plus below the card adds a new page (i.e. already does part of what we're asking for) that would affect the specifics of the approach. We wouldn't need to add a + icon, but I still think there could be improvements in styling and interface to make the process clearer.   I've update AC1 and added a Design_Feedback label as it would be good if you could weigh in on this.

**Comment by mehigh** [comment]
We could get inspired from the standard photo editors new layer icon:
![](https://pe-images.s3.amazonaws.com/basics/cs6/layers/background-layer-action/photoshop-new-layer-icon.png)
Or maybe something like this:
https://thenounproject.com/term/add-layer/340550/

This provides more information than a regular plus sign.

The plus at the bottom would make sense to add another page in the story, the layers was the only one I felt needed an improvement.

**Comment by jwold** [comment]
Dev is currently blocked by design as we need a UX suggestion for what it will look like.

**Comment by jwold** [comment]
@miina howdy! @alcurrie and I spent a bit of time going over this, and I'd love your feedback, as well as letting me know if you need anything more. 

Here is the graphic with suggested changes:

![frame](https://user-images.githubusercontent.com/3411173/48013836-1daa6a00-e0da-11e8-88a5-0d528480b23e.png)

And here is the Figma file: https://www.figma.com/file/fJFUvAKJGNjH2wCzgj5ke4PF/Untitled?node-id=1%3A127

Should I get you the new icons as SVG or PNG?

Thank you!

**Comment by jwold** [comment]
Thanks for the feedback @cathibosco, I'll go ahead and remove the UX feedback label now @alcurrie as I think this is dev ready.

**Comment by alcurrie** [comment]
@jwold  and @cathibosco  - I've pulled this back to definition, as based on further discussion on related issues, the interface sketch here, is no longer up to dated.

**Comment by alcurrie** [comment]
@cathibosco  can you take a look at this with the view to integrating it into the overall interface updates/changes. 

cc. @jwold

**Comment by cathibosco** [comment]
Got it added now - thank you @alcurrie

**Comment by mehigh** [comment]
@bluestormcreative This isn't ready for prime time. This shouldn't remove the side adder - http://cloud.urldocs.com/a4d53dd19dbf 

Instead we need to figure out a way to turn this normally hidden + button at the bottom into a page adder:
http://cloud.urldocs.com/6c65fd7a3f02

**Comment by mehigh** [comment]
Actually the hiding of those elements I did undo quickly.. and the reason the add ghosted page went berserk is due to a Gutenberg update. You were seeing a different "+" structure than I'm seeing, hence the selectors didn't play along well.
I'm updating this to work with the newest version of G and 🤞 for this not to break next week.

**Comment by mehigh** [comment]
How this looks now:
http://cloud.urldocs.com/76c76d5127ca

The small page icons in the side-nav is updated now
The last add button is transformed into a ghosted page
The page border and background color will be updated in the ticket which handles the borders.

**Comment by mehigh** [comment]
This handles the first item in AC1 and AC3. 
[x] - _Display ghosted Story page by default below the last AMP story page_ 
[ ] _On tap/click anywhere on the 'ghosted story' page a new story page is created_ -> I wasn't able to place an element with that functionality there. This work turns the old "+" at the end into a ghosted page only.
[ ] _When a new story is created_ -> no changes on this item in this ticket. Work done on the ticket which pre-populated the story page is where all of the work touching new pages is at.

AC2 - _Consider_ **not** _adding the extra page numbering_

AC 3: _Update the page icon in the block navigation_ - this is completed as well

**Comment by mehigh** [comment]
Fixed via merging #1632

**Comment by alcurrie** [comment]
Confirmed that add a new page interface, with the new 'ghosted' page interface, and the update of the icon for  pages in the block navigation elsewhere in the interface. https://cl.ly/24d18e6425f9
